CHANGELOG.md

```diff
@@ -1,335 +1,348 @@
+[#3207]: https://github.com/magento/pwa-studio/pull/3207
+[#3170]: https://github.com/magento/pwa-studio/pull/3170
+[#3130]: https://github.com/magento/pwa-studio/pull/3130
+[#3105]: https://github.com/magento/pwa-studio/pull/3105
+[#3048]: https://github.com/magento/pwa-studio/pull/3048
+[#3041]: https://github.com/magento/pwa-studio/pull/3041
+[#3049]: https://github.com/magento/pwa-studio/pull/3049
+[#3246]: https://github.com/magento/pwa-studio/pull/3246
+### Cypress tests
 
-![Custom scaffolding template](https://user-images.githubusercontent.com/4692281/112341120-25f5a680-8c8f-11eb-9174-122dcc1eb5c4.png)
+[Cypress](https://www.cypress.io/) is an end-to-end testing suite written in JavaScript.
+This release adds this framework to the PWA Studio project to increase testing automation and reduce the time spent on manual testing.
+This will enable the team to release new versions faster and more often.
 
-### MegaMenu component
+Integration tests for PageBuilder and the new Wish List feature are included in this release.
 
-A MegaMenu component is now available in the Venia UI library thanks to the contribution of community member [Marcin Kwiatkowski][].
-This component displays product categories and subcategories defined in the Magento backend.
+#### Pull Requests
 
-![MegaMenu component](https://user-images.githubusercontent.com/11998249/104089828-473f1d00-5272-11eb-900f-b7e11d9a532b.png)
+| Description                                   | PR        |
+| --------------------------------------------- | --------- |
+| Configured the project to use Cypress         | [#3082][] |
+| Added the Cypress Visual Testing plugin       | [#3136][] |
+| Added tests for the Wish List feature         | [#3146][] |
+| Added tests for PageBuilder banner component  | [#3178][] |
+| Added tests for PageBuilder buttons           | [#3194][] |
+| Added tests for PageBuilder images            | [#3195][] |
+| Added tests for multiple Wish Lists           | [#3218][] |
+| Updated the Cypress tests directory structure | [#3253][] |
 
-[Marcin Kwiatkowski]: https://github.com/Frodigo
+[#3082]: https://github.com/magento/pwa-studio/pull/3082
+[#3136]: https://github.com/magento/pwa-studio/pull/3136
+[#3146]: https://github.com/magento/pwa-studio/pull/3146
+[#3178]: https://github.com/magento/pwa-studio/pull/3178
+[#3194]: https://github.com/magento/pwa-studio/pull/3194
+[#3218]: https://github.com/magento/pwa-studio/pull/3218
+[#3253]: https://github.com/magento/pwa-studio/pull/3253
+[#3195]: https://github.com/magento/pwa-studio/pull/3195
 
-### Store switcher
+### Virtual Product types
 
-The Venia UI library now provides components that support multiple store views.
-These components let customers switch between the different store views defined in the Magento backend.
+This release contains initial work to support Virtual Product types.
+The current implementation only lets you browse and view Virtual Product types in your storefront.
 
```

Comment:
I think we should make it clearer around the issues with the work we've done. The fact they cannot be purchased and will display a notice on the product page sounds like something helpful in these notes.